### PR TITLE
Add boot performance report

### PR DESCRIPTION
**Pull request by cperciva**

### CODE REVIEW

**Comment by cperciva** [comment]
Hopefully I did this right.  I took last quarter's report and edited it...

**Review by pauamma — CHANGES_REQUESTED** [review]
Other than the inconsistent figures, LGTM.

File: 2021q4/boot-performance.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is primarily using an EC2 c5.xlarge instance as a
+reference platform and is measuring the time between when the virtual machine
+enters the EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017, and as of the end of September 2021 the FreeBSD boot
+time was reduced from approximately 30 seconds to approximately 15 seconds.
+
+During 2021Q4, a further 3 seconds of time has been shaved off the boot process,
+taking it down to roughly 9 seconds.  In addition, the userland boot process is
```
[the remark this answers is not in the document]
Comment (cperciva) [reply 2021q4/boot-performance.adoc]: The numbers don't exactly add up... the 3 seconds (actually 3.03 seconds) was from adding up the boot time eliminated by specific commits, and the change from 15 to 9 (which was actually something like 14.7 to 9.7 but I was tired when I wrote that) was measured by booting instances.

I've reworded that section now and hopefully it makes more sense.  Thanks for flagging it!

File: 2021q4/boot-performance.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Contact: Colin Percival <cperciva@FreeBSD.org>
+
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is primarily using an EC2 c5.xlarge instance as a
+reference platform and is measuring the time between when the virtual machine
+enters the EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017, and as of the end of September 2021 the FreeBSD boot
+time was reduced from approximately 30 seconds to approximately 15 seconds.
```
Comment (emaste) [inline 2021q4/boot-performance.adoc:15]: should we clarify what this includes? power on to login prompt?

**Review by pauamma — COMMENTED** [review]
LGTM now. Waiting for update (if any) following emaste's question for approval.

**Review by pauamma — APPROVED** [review]

File: 2021q4/boot-performance.adoc (hunk 2 above)
Comment (cperciva) [reply 2021q4/boot-performance.adoc:15]: The measurement is from the EC2 instance entering the "running" state to when we can SSH in.  I think "running" means roughly "CPU reset" but I'm not 100% certain (and Amazon doesn't discuss those sorts of internals).

Happy to add the detail but I'm not entirely sure that it matters; this detail is on the wiki page if anyone is interested enough to click on links.

**Pull request merged**